var/spack/repos/builtin/packages/fms/package.py

```diff
@@ -48,8 +48,13 @@ class Fms(CMakePackage):
     )
 
     variant(
-        "64bit",
+        "32bit",
         default=True,
+        description="Build a version of the library with default 32 bit reals",
+    )
+    variant(
+        "64bit",
+        default=False,
```

Comment:
Can we use a `precision` multi-valued variant, with allowed values equals to `32` and `64`, and have:
```python
conflict("precision=32,64", when="@:2022.03")
```
?

Reply:
@alalazo Actually didn't even know `conflict()` was a thing! I'm learning.

@climbfuji What do you think? I'm fine as I'm not using spack operationally yet. But this would mean you'd need to change around some variants for FMS if so. I can try prototyping it out.

Reply:
I think @alalazo's idea is great, it makes `precision` similar to the `constants` variant, and it makes handling conflicts easier. No problem to change our configs when we pull this in later.

Reply:
I think I have this now. Please take a look @alalazo